.github/workflows/reusable/security.yml

```
@@ -59,6 +59,6 @@ jobs:
       - uses: actions/setup-node@v4 # v4
         with: { node-version: ${{ inputs.node_version }}, cache: 'pnpm' }
       - uses: pnpm/action-setup@c5ba7f7862a0f64c1b1a05fbac13e0b8e86ba08c # v4
-        with: { version: 9.12.0 }
+        with: {  }
```

YAMLlint (1.38.0)

[warning] too many spaces inside empty braces

(braces)